eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/fixtures/MinimalNodeFixture.scala

```diff
@@ -71,6 +72,9 @@ object MinimalNodeFixture extends Assertions {
 
   def apply(nodeParams: NodeParams): MinimalNodeFixture = {
     implicit val system: ActorSystem = ActorSystem(s"system-${nodeParams.alias}")
+    val readyListener = TestProbe("ready-listener")
+    system.eventStream.subscribe(readyListener.ref, classOf[IAmReady])
+    waitEventStreamSynced(system.eventStream)
```

Comment:
I don't think we need that anymore now, do we? The `readyListener` will be registered before the other actors (since they're created after), so the `readyListener.expectMsgAllOf` below should be enough.